src/components/common/table.js

```diff
@@ -2,11 +2,25 @@ import React from 'react'
 import DetailText from './detail-text'
 import tableStyle from './table.module.scss'
 
+const Th = ({ children, alignLeft, scope, colSpan }) => (
+  <th
+    scope={scope}
+    colSpan={colSpan}
+    className={alignLeft ? tableStyle.alignLeft : ''}
```

Comment:
Is it possible to squelch this attribute entirely, if there’s no prop passed to the className? (Rather than rendering `class=""` in the markup.)

Reply:
Updated the ternary to a `&&`, the empty string should be squelched 👍